docs/getting_started.md

```diff
@@ -170,7 +170,7 @@ Give your app the name `guestbook`, use the project `default`, and leave the syn
 
 ![app information](assets/app-ui-information.png)
 
-Connect the [https://github.com/argoproj/argocd-example-apps.git](https://github.com/argoproj/argocd-example-apps.git) repo to Argo CD by setting repository url to the github repo url, leave revision as `HEAD`, and set the path to `guestbook`:
+Connect the [https://github.com/argoproj/argocd-example-apps.git](https://github.com/argoproj/argocd-example-apps.git) repo to Argo CD by setting repository url to the github repo url, leave revision as `HEAD`, and set the path to either `helm-guestbook` or `kustomize-guestbook`:
```

Comment:
```suggestion
Connect the [https://github.com/argoproj/argocd-example-apps.git](https://github.com/argoproj/argocd-example-apps.git) repo to Argo CD by setting repository url to the github repo url, leave revision as `HEAD`, and set the path to either `helm-guestbook` or `kustomize-guestbook`:
```

why set to `helm-guestbook` or `kustomize-guestbook`? The [guestbook](https://github.com/argoproj/argocd-example-apps/tree/master/guestbook) path exists in the repo url already.

Reply:
When I set to guestbook only it does not work. Does it work for you?

Reply:
It does not work with only guestbook. I am not able proceeding and creating the app.